tests/candev/Makefile

```diff
@@ -7,7 +7,33 @@ USEMODULE += isrpipe
 # define the CAN driver you want to use here
 CAN_DRIVER ?= native
 
-ifeq ($(CAN_DRIVER), PERIPH_CAN)
+ifeq ($(CAN_DRIVER), MCP2515)
+    USEMODULE += mcp2515
+
+	FEATURES_REQUIRED = periph_gpio periph_spi periph_gpio_irq
```

Comment:
This should not be needed because the `Makefile.dep` includes it already as a dependency

Reply:
unresolving: removing this resulted in broken can-system.